### PR TITLE
Update Solution.java

### PR DESCRIPTION
**Pull request by friselis**

### CODE REVIEW

File: introduction/welcome_to_java/Solution.java [hunk 1]
```diff
@@ -1,5 +1,4 @@
 public class Solution {
-
     public static void main(String[] args) {
```
Comment (accounts-sbeliakov) [inline introduction/welcome_to_java/Solution.java]: Please use 2 space indentation

File: introduction/welcome_to_java/Solution.java [hunk 2]
```diff
@@ -1,5 +1,4 @@
...
         /* Enter your code here. Print output to STDOUT. Your class should be named Solution. */
```
Comment (accounts-sbeliakov) [inline introduction/welcome_to_java/Solution.java]: This is a TODO, remove since it's done.
Also generally use // - style comments inside a method.

**Review by accounts-sbeliakov — COMMENTED** [review]
Good work!
Looks good, just a few comments.

File: introduction/welcome_to_java/Solution.java (hunk 1 above)
Comment (friselis) [reply introduction/welcome_to_java/Solution.java]: Done.

File: introduction/welcome_to_java/Solution.java (hunk 2 above)
Comment (friselis) [reply introduction/welcome_to_java/Solution.java]: Done.

**Comment by friselis** [comment]
Thanks!
Fixed, PTAL.

**Comment by accounts-sbeliakov** [comment]
Thanks! LGTM.

**Pull request merged**